planning/behavior_velocity_planner/src/scene_module/stop_line/scene.cpp

```diff
@@ -88,15 +88,39 @@ boost::optional<StopLineModule::SegmentIndexWithOffset> findBackwardOffsetSegmen
   return {};
 }
 
+size_t findClosestIndexWithLaneId(
```

Comment:
since lane id is necessary to find out proper lane from crossing lane find closest path index which contains stop line lane id